pkg/issuer/acme/dns/azuredns/azuredns.go

```diff
@@ -48,12 +48,12 @@ func NewDNSProvider(dns01Nameservers []string) (*DNSProvider, error) {
 	zoneName := ("AZURE_ZONE_NAME")
 	environment := ("AZURE_ENVIRONMENT")
 
-	return NewDNSProviderCredentials(environment, clientID, clientSecret, subscriptionID, tenantID, resourceGroupName, zoneName, dns01Nameservers)
+	return NewDNSProviderCredentials(environment, clientID, clientSecret, subscriptionID, tenantID, resourceGroupName, zoneName, dns01Nameservers, true)
```

Comment:
This shouldn't be hardcoded to true, and in fact, the entirety of the `NewDNSProvider` method should be removed 🙈 - can you get rid of the function altogether instead of changing this line?